master/interact.html

```diff
@@ -38,10 +38,6 @@ <h2 id="Introduction">Introduction</h2>
   attribute on the <a>'svg'</a> element and on the
   characteristics of the user agent, users are able to zoom
   into and pan around SVG content.</li>
-
```

Comment:
No reason to remove this. The `cursor` property still exists, and is still part of interactive SVG.

Reply:
How much of this do you want to keep in?

Reply:
Meh, it's pretty light-weight content anyway. `cursor` is no more essential for interactivity than `:hover` styles are. Might as well remove it. Ignore my previous comment.

Reply:
Does that mean it is approved to be merged now?